src/sentry/filters/base.py

```diff
@@ -21,11 +27,14 @@ def is_enabled(self):
             default='1' if self.default else '0',
         ) == '1'
 
-    def enable(self, status=True):
+    def enable(self, value=None):
+        if value is None:
+            value = {'active': True}
+
         ProjectOption.objects.set_value(
             project=self.project,
             key='filters:{}'.format(self.id),
-            value='1' if status else '0',
+            value='1' if value['active'] else '0',
```

Comment:
Let's make this `value.get('active', False)` to cover ourselves on a potential `KeyError`. I know I wrote this, but didn't put that much thought into it at the time since I was just explaining how it could work. :)